src/orquestra/integrations/qiskit/backend/backend.py

```diff
@@ -403,12 +409,139 @@ def _apply_readout_correction(
             meas_fitter = CompleteMeasFitter(cal_results, state_labels)
 
             # Create a measurement filter from the calibration matrix
-            self.readout_correction_filters[str(active_qubits)] = meas_fitter.filter
+            self.readout_correction_filters[str(physical_qubits)] = meas_fitter.filter
 
-        this_filter = self.readout_correction_filters[str(active_qubits)]
+        this_filter = self.readout_correction_filters[str(physical_qubits)]
         mitigated_counts = this_filter.apply(counts, method=self.noise_inversion_method)
         # round to make up for precision loss from pseudoinverses used to invert noise
         rounded_mitigated_counts = {
             k: round(v, 8) for k, v in mitigated_counts.items() if round(v, 8) != 0
         }
         return rounded_mitigated_counts
+
+
+def get_active_qubits(circuit: QuantumCircuit) -> Set[int]:
```

Comment:
This method only works for qiskit circuits, not our circuits. It these functions could be moved to a separate utils file. But I figured since we only use it here it's best to keep it here.

Reply:
It's a good place, I agree.
But "public" I didn't mean "generally applicable", I meant "would anyone want to use it outside of this module"? 
So whether it should be changed `_get_active_qubits` .
Again, I don't have a strong opinion.

Reply:
Ah! I see the confusion here. We aren't getting the the active qubits. we are getting a map from the qubits the user submitted to the qubits that are run by IBMQ. So maybe a name change to `_get_virtual_to_physical_qubits_dict` would be more descriptive?